docs/_pages/releases.md

```diff
@@ -4,15 +4,27 @@ permalink: /releases/
 layout: single
 classes: wide
 sidebar:
-  nav: "sidebar"
+nav: "sidebar"
```

Comment:
Was this change intentional?
It seems to change the generated html

Reply:
Nope, stupid mistake from my side, sorry about that